src/docs/self-hosted/troubleshooting.mdx

```diff
@@ -49,6 +48,25 @@ The _proper_ solution is as follows ([reported](https://github.com/getsentry/onp
 You can replace <code>snuba-consumers</code> with other consumer groups or <code>events</code> with other topics when needed.
 </Alert>
 
+#### Nuclear option
+The "nuclear option" is removing all Kafka-related volumes and recreating them which _will_ cause data loss. Any data that was pending there will be gone upon deleting these volumes.
+1. Stop instance:
+   ```shell
+   docker-compose stop
+   ```
+2. Remove and recreate the Kafka & Zookeeper related volumes:
+   ```shell
+   docker volume rm sentry-kafka
+   docker volume rm sentry-zookeeper
+   docker volume create --name=sentry-kafka
+   docker volume create --name=sentry-zookeeper
```

Comment:
I'll add the "Run `./install.sh` script" step below, which handles creating the volumes.

```suggestion
2. Remove the Kafka & Zookeeper related volumes:
   ```shell
   docker volume rm sentry-kafka
   docker volume rm sentry-zookeeper
```